src/Symfony/Component/Lock/Store/MongoDbStore.php

```diff
@@ -257,23 +277,35 @@ public function putOffExpiration(Key $key, float $ttl)
      */
     public function delete(Key $key)
     {
-        $this->getCollection()->deleteOne([ // filter
-            '_id' => (string) $key,
-            'token' => $this->getUniqueToken($key),
-        ]);
+        $write = new BulkWrite();
+        $write->delete(
+            [ // filter
```

Comment:
```suggestion
            [
```

This seemed self-explanatory, and you didn't have any such comments in the session handler class.

Reply:
Removed this comments that were inherited from previous implementation.